Swift Vocabulary/Swift Vocabulary/View Controllers/DefinitionViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DefinitionViewController.swift
+//  Swift Vocabulary
+//
+//  Created by James Sedlacek on 5/19/20.
+//  Copyright © 2020 James Sedlacek. All rights reserved.
+//
+
+import UIKit
+
+class DefinitionViewController: UIViewController {
+
+    var vocabWord: VocabularyWord?
+    
+    @IBOutlet weak var lblDefinition: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateViews()
+        // Do any additional setup after loading the view.
+    }
+    
+    func updateViews() {
+        if let word = vocabWord {
+            //If it is not nil, use its word and definition properties to populate the label and text view.
+            lblDefinition.text = word.definition
+        }
+    }
```

Comment:
Make sure you read the instructions carefully. You should have displayed both the word and definition in the UI.

>Add a method called updateViews().
In this method, unwrap the vocabWord property. If it is not nil, use its word and definition properties to populate the label and text view.